easybuild/tools/options.py

```diff
@@ -199,8 +201,8 @@ def override_options(self):
             'skip-test-cases': ("Skip running test cases", None, 'store_true', False, 't'),
             'umask': ("umask to use (e.g. '022'); non-user write permissions on install directories are removed",
                       None, 'store', None),
-            'optarch': ("Set architecture optimization, overriding native architecture optimizations",
-                        None, 'store', None),
+            'update-modules-tool-cache': ("Update modules tool cache file(s) after generating module file",
```

Comment:
so updating lmod cache should done separately from installing software with EB? link to the docs wrt best policy would be nice :smile: